tcl/return.tcl

```diff
@@ -50,10 +50,10 @@ proc qc::return2client { args } {
     # Content-Disposition
     if { $download } {
         # Download payload as an attachment
-        default content-disposition "attachment; filename=$filename"
+        default content-disposition "attachment; \"filename=$filename\""
```

Comment:
Okay, I'd prefer to raise an error if the $filename was problematic and enforce this restriction in the application when files are being uploaded.
Can we enforce a filename restriction in the data model ?